keps/sig-network/784-kube-proxy-component-config/README.md

```diff
@@ -300,19 +392,28 @@ No.
 
 ###### How does this feature react if the API server and/or etcd is unavailable?
 
+N/A
 
 ###### What are other known failure modes?
 
+Kube-proxy with `v1alpha2` configuration will only fail at configuration errors
+if API field `ConfigHardFail` is set to true. 
```

Comment:
That's not a failure mode of the feature, that's part of the feature.

I think "None"